package.json

```diff
@@ -45,19 +43,20 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bl": "^1.1.2",
+    "async": "^2.1.4",
+    "execa": "^0.5.0",
     "go-ipfs-dep": "0.4.4",
     "ipfs-api": "^12.0.3",
     "multiaddr": "^2.1.0",
     "once": "^1.4.0",
     "rimraf": "^2.5.4",
-    "run-series": "^1.1.4",
-    "shutdown": "^0.2.4",
-    "subcomandante": "^1.0.5"
```

Comment:
Soo even though those tests pass fine, it seems thing are wonky right now. I have 5 ipfs instances running after a single `npm test` run :( 

I will revert the switch to execa tomorrow.